rosidl_typesupport_introspection_cpp/resource/srv__type_support.cpp.em

```diff
@@ -20,15 +20,21 @@
 
 #include "rosidl_typesupport_introspection_cpp/visibility_control.h"
 
-#include "@(spec.pkg_name)/@(subfolder)/@(get_header_filename_from_msg_name(spec.srv_name))__struct.hpp"
+@{
+header_filename = get_header_filename_from_msg_name(spec.srv_name)
+if header_filename.endswith('__goal'):
+    header_filename = header_filename[:-6]
+elif header_filename.endswith('__result'):
+    header_filename = header_filename[:-8]
+elif header_filename.endswith('__feedback'):
```

Comment:
@dirk-thomas AFAIK there's no feedback service.

Reply:
Good catch - removed in 902fb0a8088c8ce50f68f93704d02cb0f3dbd073.